nodogsplash2/files/nodogsplash.config

```diff
@@ -14,27 +14,34 @@ config nodogsplash
 
   # The network the users are connected to
   option network 'lan'
-  option gatewayname 'LEDE Nodogsplash'
+  option gatewayname 'OpenWrt Nodogsplash'
   option maxclients '250'
-  option clientidletimeout '1200'
+  #Client timeouts in minutes
+  option clientidletimeout '120'
+  option clientforcetimeout '1440'
+
 
   # Your router may have several interfaces, and you
   # probably want to keep them private from the network/gatewayinterface.
   # If so, you should block the entire subnets on those interfaces, e.g.:
-  list authenticated_users 'block to 192.168.0.0/16'
-  list authenticated_users 'block to 10.0.0.0/8'
+  # list authenticated_users 'block to 192.168.0.0/16'
+  # list authenticated_users 'block to 10.0.0.0/8'
 
   # Typical ports you will probably want to open up.
-  list authenticated_users 'allow tcp port 22'
-  list authenticated_users 'allow tcp port 53'
-  list authenticated_users 'allow udp port 53'
-  list authenticated_users 'allow tcp port 80'
-  list authenticated_users 'allow tcp port 443'
+  #list authenticated_users 'allow tcp port 22'
+  #list authenticated_users 'allow tcp port 53'
+  #list authenticated_users 'allow udp port 53'
+  #list authenticated_users 'allow tcp port 80'
+  #list authenticated_users 'allow tcp port 443'
+  #Or for happy customers allow all
+  list authenticated_users 'allow all'
+
 
   # For preauthenticated users to resolve IP addresses in their
   # initial request not using the router itself as a DNS server,
-  list preauthenticated_users 'allow tcp port 53'
-  list preauthenticated_users 'allow udp port 53'
+  # Leave commented to help prevent DNS tunnelling
+  #list preauthenticated_users 'allow tcp port 53'
+  #list preauthenticated_users 'allow udp port 53'
```

Comment:
@bluewavenet how should DNS work for preauthenticated users?

Reply:
list users_to_router 'allow tcp port 53'
 list users_to_router 'allow udp port 53'
:-)

Reply:
ok